modules/terraform/cmd.go

```diff
@@ -59,3 +59,46 @@ func RunTerraformCommandE(t *testing.T, options *Options, args ...string) (strin
 		return out, retry.FatalError{Underlying: err}
 	})
 }
+
+// GetExitCodeForTerraformCommand runs terraform with the given arguments and options and return stdout/stderr.
```

Comment:
Comment doesn't seem accurate? It doesn't return stdout/stderr, it returns the exit code, right?

Reply:
updated